Sources/Datadog/RUM/DataModels/RUMDataModels.swift

```diff
@@ -0,0 +1,809 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+// This file was generated from JSON Schema using quicktype, do not modify it directly.
+
+import Foundation
+
+internal protocol RUMDataModel: Codable { }
+
+/// Schema of all properties of an Action event
+///
+/// Schema of common properties of RUM events
+// MARK: - RUMAction
+internal struct RUMAction: RUMDataModel {
```

Comment:
`struct` comment must go after section `// MARK:`, otherwise the documentation is not rendered:

<img width="731" alt="Screenshot 2020-07-23 at 09 58 41" src="https://user-images.githubusercontent.com/2358722/88264215-b11b3880-cccb-11ea-8626-1af545086110.png">

Reply:
let me see if i can change this quickly